includes/definitions/discovery/junos.yaml

```diff
@@ -154,3 +154,14 @@ modules:
                         - { value: 8, descr: offline, graph: 1, generic: 2 }
                         - { value: 9, descr: diagnostic, graph: 1, generic: 3 }
                         - { value: 10, descr: standby, graph: 1, generic: 3 }
+                -
+                    oid: jnxVirtualChassisMemberTable
+                    value: jnxVirtualChassisMemberRole
+                    num_oid: '.1.3.6.1.4.1.2636.3.40.1.4.1.1.1.3.{{ $index }}'
+                    descr: jnxVirtualChassisMemberSerialnumber
```

Comment:
Maybe instead of Serialnumber as descr you could use another one more identificative? Just a question more than a review. Maybe `jnxVirtualChassisMemberAlias`?

Reply:
Hi @TheGreatDoc, I decided on jnxVirtualChassisMemberSerialnumber as from a replacement perspective when a remote FE is on-site to replace a device, there's nothing clearer than the serial number with which to refer against. Also, it isn't always guaranteed that the EX has the jnxVirtualChassisMemberAlias set whereas the serial number is always present.